indexer/test/indexer.test.js

```diff
@@ -905,6 +954,28 @@ describe('POST /service-provider-registry/provider-removed', () => {
   })
 })
 
+async function withDataSet(
+  env,
+  { dataSetId = randomId(), withCDN = true, payer, payee },
+) {
+  await env.DB.prepare(
+    `
+    INSERT INTO data_sets (
+      id,
+      with_cdn,
+      payer,
+      payee
+    )
+    VALUES (?, ?, ?, ?)
+    ON CONFLICT DO NOTHING
+  `,
+  )
+    .bind(String(dataSetId), withCDN ? 1 : 0, payer, payee)
```

Comment:
I think this can be simplified as follows:

```suggestion
    .bind(String(dataSetId), withCDN, payer, payee)
```